src/test/smoke/features/smoke-test.feature

```diff
@@ -3,7 +3,7 @@ Feature: Smoke test
   As the continuous delivery pipeline
   I want know that a new release has deployed correctly and is functional
 
-  Scenario: The overview page is available and starts the application process
-    Given I navigate to the Apply for Healthy Start overview page
+  Scenario: The guidance page is available and starts the application process
```

Comment:
I think this should start `The apply guidance page is available...`

Reply:
Updated